docs/2.guide/1.concepts/9.code-style.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Code Style'
+description: "Nuxt supports ESLint out of the box"
+---
+
+## ESLint
+
+The recommended approach for Nuxt is to enable ESLint support using [`@nuxt/eslint-config`](https://github.com/nuxt/eslint-config). 
+
+**Install**
+
+  ```bash [yarn]
+  yarn add --dev eslint @nuxt/eslint-config
+  ```
+
+  ```bash [npm]
+  npm install --save-dev eslint @nuxt/eslint-config
+  ```
+
+  ```bash [pnpm]
+  pnpm add -D eslint @nuxt/eslint-config
+  ```
+
+  ```bash [bun]
+  bun add -D eslint @nuxt/eslint-config
+  ```
+
+**Add `.eslintrc.cjs` to the root folder of your Nuxt app**
+
+```js
+module.exports = {
+  root: true,
+  extends: ['@nuxt/eslint-config'],
+}
+```
+
+## Modify package.json
+
+Add the below to lint commands to your `package.json` script section:
+
+```json
+  "scripts": {
+    ...
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    ...
+  },
+```
+
+Run the lint command to check if the code style is correct or run `lint:fix` to automatically fix issues.
+
+## Configuring VSCode
+
+Install the [VSCode ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
+
+In VSCode press `ctrl+shift+p` (`cmd+shift+p` on Mac) to open the command prompt, find `Open Workspace Settings (JSON)`, add the below lines to the JSON and save:
+
+```json
+{
+  "editor.formatOnSave": true,
```

Comment:
I am not sure if enabling this has something to do with ESLint (`codeActionsOnSave` is different from `formatOnSave`, I believe). In my own usage, I set this off to disable Prettier in favor of ESLint's auto-fix. I guess for `@nuxt/eslint-config`, we don't need to mention this line as there is no formatting involved.

```suggestion
```